conans/test/integration/toolchains/meson/test_mesontoolchain.py

```diff
@@ -517,3 +518,26 @@ def generate(self):
     client.save({"conanfile.py": conanfile})
     client.run("install .", assert_error=True)
     assert "You can only pass native=True if you're cross-building" in client.out
+
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="requires Win")
+def test_compiler_path_with_spaces():
+    conanfile = (GenConanfile().with_name("lib").with_version("1.0").with_generator("MesonToolchain")
+                 .with_settings("compiler"))
+    profile = textwrap.dedent("""
+        [settings]
+        arch=x86
+        os=Windows
+        compiler=msvc
+        compiler.version=193
+        compiler.runtime=static
+        build_type=Release
+        [conf]
+        tools.build:compiler_executables={"c":"c compiler path with spaces", "cpp":"cpp compiler path with spaces"}
```

Comment:
you can just compose the default profile with the conf or pass the conf via the cli, no need to pass the full hardcoded profile